src/JustEat.StatsD/JustEat.StatsD.csproj

```diff
@@ -15,4 +21,7 @@
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.0.0" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="1.1.0" />
```

Comment:
Why the additional reference for all TFMs of `1.1.0` when there's a `2.0.0` on the line above that would be enabled if the `Condition` was removed?